lib/qubinode_ocp4.sh

```diff
@@ -330,15 +330,24 @@ deploy_compute_nodes () {
 
 
 wait_for_ocp4_nodes_shutdown () {
-  build_ocp4_vm_list
-  for vm in ${all_vms[@]}
+  i="$(sudo virsh list | grep running | grep master |wc -l)"
+
+  while [ $i -ne 0 ]
   do
-      isvmRunning | while read VM
```

Comment:
What was wrong with the isvmRunning function? How come you didn't just improve on that function?